### DISCUSSION TITLE
CRenderFunction

### DISCUSSION THREAD

**Discussion by nbmartino** [coreui/coreui-free-vue-admin-template#290] [Q&A]
Hi,
Is the CRenderFunction accessible in free-vue-admin template?
I'm trying create a dynamic sidebar through Composition API approach, is it possible?

Reference: https://codesandbox.io/s/coreui-vue-dynamic-sidebar-gurry?file=/src/App.vue

Thank you in advance!

**Comment by pervej-fac** [comment]
Code released under [the MIT license](https://github.com/coreui/coreui-free-react-admin-template/blob/master/LICENSE). There is only one limitation you can't can’t re-distribute the CoreUI as stock. You can’t do this if you modify the CoreUI. In past we faced some problems with persons who tried to sell CoreUI based templates.

here is shown, we can not re-distribute. what is meant by this?

Suppose I have used the dashboard in my sales management software and others. and Can i sell the softwares? Have any restriction by this re-distribution issues?
